torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -1322,6 +1323,16 @@ def run(
         benchmark_run=False,
         **kwargs,
     ):  # type:ignore[override]
+        debug_mode = get_active_debug_mode()
```

Comment:
i wonder if it's possible to move this to `CachingAutotuner` class to further reduce the overhead, thinking....